portfolio/src/main/java/com/google/sps/servlets/OfficeGame.java

```diff
@@ -72,20 +91,30 @@ public String getAnswer() {
   public void submitAnswer(String guess) {
     lastSubmission = values.get(currentPrompt).equals(guess);
     if(lastSubmission){
-        score += 1;
+      score += 1;
```

Comment:
Do you need to use the datastore for this? High score is just a single number right? Now you're storing in your store a list of contiguous integers (since high score will always increase by 1 each time). You probably only need to use the datastore when you want to store the highScore for each user (where the data will be [<userId1, highScore>, <userId2, highScore>].

Reply:
Yes it is true I just want to store a single integer, but what is the alternative to doing that other than Datastore?

Reply:
You can just use the existing variable `highScore`?